### ISSUE TITLE
[java] InvalidLogMessageFormat: Lombok @Slf4j annotation is not interpreted by PMD

### ISSUE THREAD

**Issue by oowekyala** [pmd/pmd#5702]
**Rule**: [InvalidLogMessageFormat](https://docs.pmd-code.org/latest/pmd_rules_java_errorprone.html#invalidlogmessageformat)

> Hi, new to this project, just wondering if supporting lombok in `InvalidLogMessageFormat` falls into the scope of this issue?
> 
> 
> 
> For example, pmd reports 'missing arguments' on
> 
> 
> 
> ```java
> 
> public class Book {
> 
>     private static final Logger LOGGER = LoggerFactory.getLogger(Book.class);
> 
> 
> 
>     public void logMe() {
> 
>         LOGGER.info("Hello, %s!");
> 
>     }
> 
> }
> 
> ```
> 
> ...but not on
> 
> ```java
> 
> @Slf4j
> 
> public class Book {
> 
>     public void logMe() {
> 
>         log.info("Hello, %s!");
> 
>     }
> 
> }
> 
> ```
> 
> Thanks :) 

 _Originally posted by @Frederick888 in [#3119](https://github.com/pmd/pmd/issues/3119#issuecomment-874421359)_
